package.json

```diff
@@ -13,5 +13,8 @@
   "repository": {
     "type" : "git",
     "url" : "https://github.com/cfn-modules/route53-hosted-zone-wrapper.git"
```

Comment:
is bundleDependencies missing?

Reply:
Can we .gitignore the node_modules folder?

Reply:
@michaelwittig `bundledDependencies` has already been part of the package.json in line 7. Instead of fetching the dependency via a git module, we are now using a npm module reference.

Reply:
> Can we .gitignore the node_modules folder?

Oh, yes!

Reply:
> @michaelwittig `bundledDependencies` has already been part of the package.json in line 7. Instead of fetching the dependency via a git module, we are now using a npm module reference.

oh, I see